reference/docs-conceptual/getting-started/fundamental/Understanding-the-PowerShell-Pipeline.md

```diff
@@ -0,0 +1,117 @@
+Get-ChildItem -Path C:\WINDOWS\System32 | Out-Host -Paging
+```
+
+```Output
+    Directory: C:\WINDOWS\system32
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+d-----        4/12/2018   2:15 AM                0409
+d-----        5/13/2018  11:31 PM                1033
+d-----        4/11/2018   4:38 PM                AdvancedInstallers
+d-----        5/13/2018  11:13 PM                af-ZA
+d-----        5/13/2018  11:13 PM                am-et
+d-----        4/11/2018   4:38 PM                AppLocker
+d-----        5/13/2018  11:31 PM                appmgmt
+d-----        7/11/2018   2:05 AM                appraiser
+d---s-        4/12/2018   2:20 AM                AppV
+d-----        5/13/2018  11:10 PM                ar-SA
+d-----        5/13/2018  11:13 PM                as-IN
+d-----        8/14/2018   9:03 PM                az-Latn-AZ
+d-----        5/13/2018  11:13 PM                be-BY
+d-----        5/13/2018  11:10 PM                BestPractices
+d-----        5/13/2018  11:10 PM                bg-BG
+d-----        5/13/2018  11:13 PM                bn-BD
+d-----        5/13/2018  11:13 PM                bn-IN
+d-----        8/14/2018   9:03 PM                Boot
+d-----        8/14/2018   9:03 PM                bs-Latn-BA
+d-----        4/11/2018   4:38 PM                Bthprops
+d-----        4/12/2018   2:19 AM                ca-ES
+d-----        8/14/2018   9:03 PM                ca-ES-valencia
+d-----        5/13/2018  10:46 PM                CatRoot
+d-----        8/23/2018   5:07 PM                catroot2
+<SPACE> next page; <CR> next line; Q quit
+...
+```
+
+Paging also reduces CPU utilization because processing transfers to the `Out-Host` cmdlet when it
+has a complete page ready to display. The cmdlets that precede it in the pipeline pause execution
+until the next page of output is available.
+
+You can see the difference Windows Task Manager to monitor CPU and memory used by PowerShell. Run
+the following command: `Get-ChildItem C:\\Windows -Recurse`. Compare the CPU and memory usage to
+this command: `Get-ChildItem C:\\Windows -Recurse | Out-Host -Paging`.
+
+## Objects in the pipeline
+
+When you run a cmdlet in PowerShell, you see text output because it is necessary to represent
+objects as text in a console window. The text output may not display all of the properties of the object being output.
+
+For example, consider the `Get-Location` cmdlet. If you run `Get-Location` while
+your current location is the root of the C drive, you see the following output:
+
+```
+PS> Get-Location
+
+Path
+----
+C:\
+```
+
+The text output is a summary of information, not a complete representation of the object returned
+by `Get-Location`. The heading in the output is added by the process that formats the data for
+onscreen display.
+
+When you pipe the output to the `Get-Member` cmdlet you get information about the object returned by
+`Get-Location`.
+
+```
```

Comment:
Looks like this should be a PowerShell code-fence.

Reply:
Split the output and added fences.